docs/source/faq.rst

```diff
@@ -167,4 +167,6 @@ There is also `BATS_TEST_SKIPPED` which will be non-empty (contains the skip mes
 How can I setup/cleanup before/after all tests?
 -----------------------------------------------
 
-Currently, this is not supported. Please contribute your usecase to issue `#39 <https://github.com/bats-core/bats-core/issues/39>`_.
+Setup/cleanup before/after all tests can be achieved using the special `setup-suite` and `teardown-suite` functions.
+These functions must be placed into a dedicated `setup_suite.bash` file next to your `.bats` files. 
+For more information check out the `setup` and `teardown` section under `Writing Tests`.
```

Comment:
Please link to the sections.

Reply:
@martin-schulze-vireso, do you mean to link to the part in the documentation? I am not sure how to do this, since I assume the documentation is processed by an `rst` to `html` converter? What URL should I use?

I originally wanted to do that but did not find any other example for a intra-doc cross-reference. Could you maybe point one out for me?

Reply:
To be honest, I don't know either. Have a Look here https://www.sphinx-doc.org/en/master/usage/referencing.html

Reply:
It appears to be a special case, because the document I am linking from is ReST, but the document I am linking to is a Markdown file. 
I found [this documentation](https://docs.readthedocs.io/en/stable/guides/cross-referencing-with-sphinx.html#automatically-label-sections), but I an not certain how special characters are used in the cross-reference definition.